src/main/groovy/com/sysgears/grain/highlight/CachedHighlighter.groovy

```diff
@@ -19,9 +19,8 @@ package com.sysgears.grain.highlight
 import com.sysgears.grain.annotations.Uncached
 import com.sysgears.grain.config.Config
 import com.sysgears.grain.taglib.GrainUtils
-import com.sysgears.grain.taglib.Site
+import org.jetbrains.annotations.Nullable
```

Comment:
Why to use JetBrains annotations instead of Javax, they look vendor-specific. Why do you say, that javax annotations are obsolete?